packages/installWatson.sh

```diff
@@ -11,14 +11,22 @@ source "$PACKAGE_HOME/util.sh"
 
 echo Installing Watson package.
 
-createPackage watson \
-    -a description "Actions for the Watson analytics APIs" \
-    -a parameters '[ {"name":"username", "required":false}, {"name":"password", "required":false, "type":"password"} ]'
+createPackage watson-translate \
+    -a description "Actions for the Watson analytics APIs to translate" \
+    -a parameters '[ {"name":"bluemixServiceName", "required":false, "bindTime":true}, {"name":"username", "required":false}, {"name":"password", "required":false, "type":"password"} ]'
+
+createPackage watson-speechToText \
+    -a description "Actions for the Watson analytics APIs to convert speech into text" \
+    -a parameters '[ {"name":"bluemixServiceName", "required":false, "bindTime":true}, {"name":"username", "required":false}, {"name":"password", "required":false, "type":"password"} ]'
+
+createPackage watson-textToSpeech \
+    -a description "Actions for the Watson analytics APIs to convert text into speech" \
+    -a parameters '[ {"name":"bluemixServiceName", "required":false, "bindTime":true}, {"name":"username", "required":false}, {"name":"password", "required":false, "type":"password"} ]'
 
 waitForAll
 
-install "$PACKAGE_HOME/watson/speechToText.js" \
-    watson/speechToText \
+install "$PACKAGE_HOME/watson/watson-speechToText/speechToText.js" \
```

Comment:
add package parameter 
add the package parameter 
`-p bluemixServiceName speech_to_tex`